### ISSUE TITLE
Get import/named errors when using TypeScript

### ISSUE THREAD

**Issue by odanado** [nuxt/eslint-config#169]
https://github.com/odan-sandbox/nuxt-eslint-error-sandbox/blob/main/plugins/my.ts

I get an `import/named` error when I run eslint for the following code.
```typescript
import Vue, { PropType } from 'vue'
```

```
/Users/odan/source/github.com/odan-sandbox/nuxt-eslint-error-sandbox/plugins/my.ts
   3:15  error    PropType not found in 'vue'   import/named
  25:53  warning  Unexpected console statement  no-console
```

The .eslintrc.js that I used is as follows This is unchanged from what `yarn create-app nuxt` generated.
```js
module.exports = {
  root: true,
  env: {
    browser: true,
    node: true,
  },
  extends: [
    '@nuxtjs/eslint-config-typescript',
    'plugin:nuxt/recommended',
    'prettier',
  ],
  plugins: [],
  // add your custom rules here
  rules: {},
}
```

This problem can be solved by setting `import/parsers` and `import/resolver`.
ref: https://github.com/odan-sandbox/nuxt-eslint-error-sandbox/blob/main/.eslintrc.with-resolver.js

I believe that these settings should be added to https://github.com/nuxt/eslint-config/blob/master/packages/eslint-config-typescript/index.js.

**Comment by yshrsmz** [comment]
I think you can also workaround this by splitting import statement

```
import Vue from 'vue'
import type { PropType } from 'vue'
```
